config/travis.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
   "canvasPoller": {
```

Comment:
Note that there's a number of failures in the tests now:
https://travis-ci.org/ets-berkeley-edu/collabosphere/builds/143465746